pkg/eventqueue/eventqueue.go

```diff
@@ -68,6 +67,9 @@ type EventQueue struct {
 	// name is used to differentiate this EventQueue from other EventQueues that
 	// are also running in logs
 	name string
+
```

Comment:
I don't understand why this commit is needed or what is trying to solve.

Reply:
Added more context to the commit message.